lua/obsidian/command.lua

```diff
@@ -95,7 +95,11 @@ command.open = function(client, data)
   local encoded_path = util.urlencode(tostring(path))
 
   local app = "/Applications/Obsidian.app"
-  if Path:new(app):exists() then
+  local sysname = vim.loop.os_uname().sysname
+  if sysname == "Linux" then
+    local cmd = ("xdg-open 'obsidian://open?vault=%s&file=%s'"):format(encoded_vault, encoded_path)
+    vim.fn.jobstart(cmd)
+  elseif Path:new(app):exists() then
```

Comment:
Probably should also check for Darwin here:
```suggestion
  elseif sysname == "Darwin" and Path:new(app):exists() then
```